modules/components/pages/redpanda/about.adoc

```diff
@@ -2,16 +2,43 @@
 
 component_type_dropdown::[]
 
-As well as the xref:components:logger/about.adoc[default logger], you can configure Redpanda Connect to send logs to a topic in a Redpanda cluster.
+// tag::single-source[]
 
-The configuration for this feature lives under the `redpanda` namespace, with the following default values:
+// © 2024 Redpanda Data Inc.
 
-[source,yaml]
-----
+As well as the xref:components:logger/about.adoc[default logger], you can configure Redpanda Connect to send logs and status updates to topics on a Redpanda cluster.
+
+The `redpanda` namespace contains the configuration of the logging server, which uses the following default values:
+
+[tabs]
+======
+Common::
++
+--
+
+```yml
+# Common configuration fields, showing default values
 redpanda:
-  seed_brokers: []
-  logs_topic: __redpanda.connect.logs
+  pipeline_id: ""
+  seed_brokers: [] # No default (optional)
+  logs_topic: ""
   logs_level: info
+  status_topic: ""
+```
+
+--
+Advanced::
++
+--
+
+```yml
+# All configuration fields, showing default values
+redpanda:
+  pipeline_id: ""
+  seed_brokers: [] # No default (optional)
+  logs_topic: ""
+  logs_level: info
+  status_topic: ""
   client_id: benthos
```

Comment:
Should the `client_id` still be `benthos`?

Reply:
It currently defaults to `benthos` still, we'll eventually update that but for now I would leave it.